docs/tutorials/introductory_tutorials/traffic_prediction.jl

```diff
@@ -0,0 +1,246 @@
+
+# ╔═╡ 3d5503bc-bb97-422e-9465-becc7d3dbe07
+size(g.node_data.targets[1])
+
+# ╔═╡ 3569715d-08f5-4605-b946-9ef7ccd86ae5
+md"
+In the case of the targets the first dimension is 1 because they store just the speed value.
+"
+
+# ╔═╡ aa4eb172-2a42-4c01-a6ef-c6c95208d5b2
+g.node_data.features[1][:,1,:]
+
+# ╔═╡ 367ed417-4f53-44d4-8135-0c91c842a75f
+g.node_data.features[2][:,1,:]
+
+# ╔═╡ 7c084eaa-655c-4251-a342-6b6f4df76ddb
+g.node_data.targets[1][:,1,:]
+
+# ╔═╡ bf0d820d-32c0-4731-8053-53d5d499e009
+function plot_data(data,sensor)
+	p = plot(legend=false, xlabel="Time (h)", ylabel="Normalized speed")
+	plotdata = []
+	for i in 1:3:length(data)
+		push!(plotdata,data[i][1,sensor,:])
+	end
+	plotdata = reduce(vcat,plotdata)
+	plot!(p, collect(1:length(data)), plotdata, color = :green, xticks =([i for i in 0:50:250], ["$(i)" for i in 0:4:24]))
+	return p
+end
+
+# ╔═╡ cb89d1a3-b4ff-421a-8717-a0b7f21dea1a
+plot_data(g.node_data.features[1:288],1)
+
+# ╔═╡ 3b49a612-3a04-4eb5-bfbc-360614f4581a
+md"
+Now let's construct the static graph, the temporal features and targets from the dataset.
+"
+
+# ╔═╡ 95d8bd24-a40d-409f-a1e7-4174428ef860
+begin
+	graph = GNNGraph(g.edge_index; edata = g.edge_data, g.num_nodes)
+	features = g.node_data.features
+	targets = g.node_data.targets
+end;  
+
+# ╔═╡ fde2ac9e-b121-4105-8428-1820b9c17a43
+md"
+Now let's construct the `train_loader` and `data_loader`.
+"
+
+
+# ╔═╡ 111b7d5d-c7e3-44c0-9e5e-2ed1a86854d3
+begin
+	train_loader = zip(features[1:200], targets[1:200])
+	test_loader = zip(features[2001:2288], targets[2001:2288])
+end;
+
+# ╔═╡ 572a6633-875b-4d7e-9afc-543b442948fb
+md"
+## Model: T-GCN
+
+We use the T-GCN model from the paper [T-GCN: A Temporal Graph Convolutional Network for Traffic Prediction] (https://arxiv.org/pdf/1811.05320.pdf), which consists of a graph convolutional network (GCN) and a gated recurrent unit (GRU). The GCN is used to capture spatial features from the graph, and the GRU is used to capture temporal features from the feature time series.
+"
+
+# ╔═╡ 5502f4fa-3201-4980-b766-2ab88b175b11
+model = GNNChain(TGCN(2 => 100), Dense(100, 1))
+
+# ╔═╡ 4a1ec34a-1092-4b4a-b8a8-bd91939ffd9e
+md"
+![](https://www.researchgate.net/profile/Haifeng-Li-3/publication/335353434/figure/fig4/AS:851870352437249@1580113127759/The-architecture-of-the-Gated-Recurrent-Unit-model.jpg)
+"
+
+# ╔═╡ 755a88c2-c2e5-46d1-9582-af4b2c5a6bbd
+md"
+## Training
+
+We train the model for 100 epochs, using the Adam optimizer with a learning rate of 0.001. We use the mean absolute error (MAE) as the loss function.
+"
+
+# ╔═╡ e83253b2-9f3a-44e2-a747-cce1661657c4
+function train(graph, train_loader, model)
+
+    opt = Flux.setup(Adam(0.001), model)
+
+    for epoch in 1:100
+        for (x, y) in train_loader
+            x, y = (x, y)
+            grads = Flux.gradient(model) do model
+                ŷ = model(graph, x)
+                Flux.mae(ŷ, y) 
+            end
+            Flux.update!(opt, model, grads[1])
+		end
```

Comment:
still the indentation is wrong.

Reply:
In my editor and documentation it shows correctly, I changed the github setting to tab size 4 and I can see it right here as well.